src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
 }
+
+/* #region HEADER */
+
+@mixin flex-center($justify: space-between, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
```

Comment:
It's recommended to name mixins in a more descriptive way. The name `flex-center` might suggest that it's used to center items, but it actually sets up a flex container with customizable `justify-content` and `align-items`. Consider renaming it to something like `flex-setup`.